dashboard_test.go

```diff
@@ -22,11 +22,12 @@ func TestView_MarshalJSON(t *testing.T) {
 		wants wants
 	}{
 		{
+			name: "xy",
```

Comment:
:question: Is there a reason we are updating tests for `xy` in this pr? This seems a bit out of scope for geo.

Reply:
Yes, I did it because I have been adding extra test case specific to geo as requested. I wanted to make the source file to be more readable and make it obvious what are the individual cases testing. This individual addition means that you will get 'xy' or 'geo' serialization failed message if something goes wrong.